### ISSUE TITLE
Issue null cannot be cast to non-null type kotlin.String

### ISSUE THREAD

**Issue by stact** [matanshukry/flutter_google_places_sdk#25]
Hi

On android seems to have an issue. Used with physical phone

To reproduce
https://github.com/matanshukry/flutter_google_places_sdk/tree/master/flutter_google_places_sdk/example

```log
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): Failed to handle method call
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): java.lang.NullPointerException: null cannot be cast to non-null type kotlin.String
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at com.msh.flutter_google_places_sdk.FlutterGooglePlacesSdkPlugin.readLocale(FlutterGooglePlacesSdkPlugin.kt:406)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at com.msh.flutter_google_places_sdk.FlutterGooglePlacesSdkPlugin.onMethodCall(FlutterGooglePlacesSdkPlugin.kt:47)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:262)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$io-flutter-embedding-engine-dart-DartMessenger(DartMessenger.java:319)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source:12)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at android.os.Handler.handleCallback(Handler.java:883)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at android.os.Handler.dispatchMessage(Handler.java:100)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at android.os.Looper.loop(Looper.java:214)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at android.app.ActivityThread.main(ActivityThread.java:7397)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at java.lang.reflect.Method.invoke(Native Method)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(32468): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:935)
```

**Comment by 404krishna2** [comment]
Hi guys,

The same problem exists on Android devices.


E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): java.lang.NullPointerException: null cannot be cast to non-null type kotlin.String
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at com.msh.flutter_google_places_sdk.FlutterGooglePlacesSdkPlugin.readLocale(FlutterGooglePlacesSdkPlugin.kt:406)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at com.msh.flutter_google_places_sdk.FlutterGooglePlacesSdkPlugin.onMethodCall(FlutterGooglePlacesSdkPlugin.kt:47)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:262)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$DartMessenger(DartMessenger.java:319)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at io.flutter.embedding.engine.dart.-$$Lambda$DartMessenger$TsixYUB5E6FpKhMtCSQVHKE89gQ.run(Unknown Source:12)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at android.os.Handler.handleCallback(Handler.java:883)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at android.os.Handler.dispatchMessage(Handler.java:100)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at android.os.Looper.loop(Looper.java:238)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at android.app.ActivityThread.main(ActivityThread.java:7824)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at java.lang.reflect.Method.invoke(Native Method)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
E/MethodChannel#plugins.msh.com/flutter_google_places_sdk(21358): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:981)



I/flutter (21358): FlutterGooglePlacesSdk::_ensureInitialized error: PlatformException(error, null cannot be cast to non-null type kotlin.String, null, java.lang.NullPointerException: null cannot be cast to non-null type kotlin.String
I/flutter (21358): 	at com.msh.flutter_google_places_sdk.FlutterGooglePlacesSdkPlugin.readLocale(FlutterGooglePlacesSdkPlugin.kt:406)
I/flutter (21358): 	at com.msh.flutter_google_places_sdk.FlutterGooglePlacesSdkPlugin.onMethodCall(FlutterGooglePlacesSdkPlugin.kt:47)
I/flutter (21358): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:262)
I/flutter (21358): 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
I/flutter (21358): 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$DartMessenger(DartMessenger.java:319)
I/flutter (21358): 	at io.flutter.embedding.engine.dart.-$$Lambda$DartMessenger$TsixYUB5E6FpKhMtCSQVHKE89gQ.run(Unknown Source:12)
I/flutter (21358): 	at android.os.Handler.handleCallback(Handler.java:883)
I/flutter (21358): 	at android.os.Handler.dispatchMessage(Handler.java:100)
I/flutter (21358): 	at android.os.Looper.loop(Looper.java

**Comment by stact** [comment]
Seems the package is not maintained anymore =)

**Comment by matanshukry** [comment]
* Fixed in android 0.1.2+4 - https://pub.dev/packages/flutter_google_places_sdk_android/versions/0.1.2+4
* Also updated sdk (main) package to latest android and ios versions - flutter_google_places_sdk: ^0.3.2+3
